OptimoveSDK/Sources/Classes/Legacy/Logger/OptiLoggerOutputStream.swift

```diff
@@ -3,7 +3,7 @@
 import Foundation
 
 /// Receiver of logs of OptimoveSDK.
-@objc public protocol OptiLoggerOutputStream {
+public protocol OptiLoggerOutputStream {
```

Comment:
Any. reason we're removing objc annotations here? (presume no direct consumers so it's not necessary)

Reply:
I am trying to remember the exact reason for removing it, but cannot. In any case, it is used for QA app access for internal logs -- it does not affect SDK usage for tenants.